src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -862,3 +864,32 @@ OCC::Result<OCC::Vfs::ConvertToPlaceholderResult, QString> OCC::CfApiWrapper::co
         return stateResult;
     }
 }
+
+OCC::Result<OCC::Vfs::ConvertToPlaceholderResult, QString> OCC::CfApiWrapper::updatePlaceholderMarkInSync(const QString &path, const QByteArray &fileId, const QString &replacesPath)
```

Comment:
@mgallien I would make it one function as it only differs by the absence of `CF_FS_METADATA metadata;`, or maybe split it into 2 smaller functions, and use a parameter to control the logic. Now, 2 methods are almost the same.

Reply:
done